Automattic/wp-calypso
# Woo Onboarding: Add revenue dropdown

#### Changes proposed in this Pull Request

* Adds revenue dropdown to business-info step.

#### Testing instructions



* Navigate to `http://calypso.localhost:3000/start/woocommerce-install/store-address?site=<siteId>`
* Fill up the fields specially the country/state one, select US - <Any State> and click continue
* On `Currently selling elsewhere? (optional)` dropdown select "Yes on another platform"
* Revenue Dropdown should appear and contain the same options as the woocommerce one.
* To try out different currencies, go back to the address step and select a different country.



Related to #59837
